src/autoscaler/eventgenerator/client/log_cache_client.go

```diff
@@ -194,26 +296,13 @@ func logCacheFiltersFor(endTime time.Time, metricType string) (readOptions []log
 		readOptions = append(readOptions, logcache.WithNameFilter("disk"))
 	case models.MetricNameDiskUtil:
 		readOptions = append(readOptions, logcache.WithNameFilter("disk|disk_quota"))
-	case models.MetricNameResponseTime, models.MetricNameThroughput:
-		readOptions = append(readOptions, logcache.WithNameFilter("http"))
 	default:
 		readOptions = append(readOptions, logcache.WithNameFilter(metricType))
 	}
 
 	return readOptions
 }
 
-func getEnvelopeType(metricType string) rpc.EnvelopeType {
```

Comment:
`getEnvelopeType` is no longer required since we only read `logcache_v1.EnvelopeType_GAUGE` after migrating `throughput` and `responsetime` to be retrieved via PromQL API